pkgs/by-name/mu/multiqc/package.nix

```
@@ -63,19 +65,15 @@ python3Packages.buildPythonApplication rec {
     typeguard
     tqdm
     python-dotenv
-    natsort
-    tiktoken
     jsonschema
-    polars
-    pyarrow
   ];
 
   optional-dependencies = {
     dev = with python3Packages; [
       pre-commit-hooks
       pdoc3
       pytest
-      pytest-cov
+      pytest-cov-stub
```

we try not to propagate the stub to downstream users
```suggestion
      pytest-cov
```